### PR TITLE
Update Teams Events Policy to include the AllowEmailEditing policy

### PR DESCRIPTION
**Pull request by navirdee**
Updated Teams Events Policy to include the AllowEmailEditing policy as well.

### CODE REVIEW

File: teams/teams-ps/teams/New-CsTeamsEventsPolicy.md [hunk 1]
```diff
@@ -67,6 +67,26 @@ Possible values are:
...
+
+Required: True
+Position: Named
+Default value: Enabled
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -AllowEmailEditing
+This setting governs if a user is allowed to edit the communication emails in Teams Town hall or Webinar Events.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/New-CsTeamsEventsPolicy.md:83]:
```suggestion
This setting governs if a user is allowed to edit the communication emails in Teams Town Hall or webinar events. 

```

File: teams/teams-ps/teams/Set-CsTeamsEventsPolicy.md [hunk 2]
```diff
@@ -119,6 +119,56 @@ Accept wildcard characters: False
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -AllowEmailEditing
+This setting governs if a user is allowed to edit the communication emails in Teams Town hall or Webinar Events.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Set-CsTeamsEventsPolicy.md:135]:
```suggestion
This setting governs if a user is allowed to edit the communication emails in Teams Town Hall or webinar events.

```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @navirdee, kindly commit the suggested changes. Many thanks.

**Review by get-itips — APPROVED** [review]

File: teams/teams-ps/teams/New-CsTeamsEventsPolicy.md (hunk 1 above)
Comment (navirdee) [reply teams/teams-ps/teams/New-CsTeamsEventsPolicy.md:83]: @JohanFreelancer9 we cannot keep the 'w' as lower case as Webinar itself is an independent product similar to Teams Town hall. So not changing it to lower case

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by shivtorov** [comment]
pls consider adding description for the remaining params as well:
- AllowedQuestionTypesInRegistrationForm
- AllowedTownhallTypesForRecordingPublish
- AllowedWebinarTypesForRecordingPublish
- AllowEventIntegrations
- TownhallChatExperience

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**